OpenRA.Mods.Common/Traits/Power/CanPowerDown.cs

```diff
@@ -61,6 +58,12 @@ protected override void TraitEnabled(Actor self)
 			power.UpdateActor(self);
 		}
 
+		protected override void TraitResumed(Actor self)
```

Comment:
What is the purpose of `TraitResumed` here? I think we want to modify the behaviour only on disabling/enabling? Also this seems to work well without this method?

Reply:
> What is the purpose of TraitResumed here?

When the trait becomes unpaused (resumed), we want it to be 'active' again, if I'm not mistaken.

> Also this seems to work well without this method?

Then something else probably does *not* work as intended.

Reply:
My gut feeling is that both `TraitResumed` and `TraitEnabled` may be able to go away here.

When we disable the trait we force `isPoweredDown = False` and need to revoke the condition, so the `Update` is needed.  When we resume the trait we know that `isPoweredDown` is *still* false, so shouldn't need to do anything.  When we pause we lock `isPoweredDown` to its current value, which will also not change on Resume.